### PR TITLE
Replaced the * with XYZ

### PR DESCRIPTION
**Pull request by Kair0s3**

### CODE REVIEW

**Review by sean-phang — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ The `Logic` component consists of:
 
 - `Parser`: Handles user input and decides the `Command` object to create.
 - `RecipeCommandParser`: When `Parser` detects that a `RecipeXYZComand` object needs to be created, the user input is handed over from `Parser` and will decide the `Command` object to create.
-- `Command`: An abstract class that defines the blueprint for the derived `*Command` classes. <a name="designListOfCommands"></a>
+- `Command`: An abstract class that defines the blueprint for the derived `XYZCommand` classes. <a name="designListOfCommands"></a>
```
Comment (rizemon) [inline docs/DeveloperGuide.md:182]: For `RecipeCommandParser` line, change `Command` to `RecipeXYZCommand` and help fix the type for `RecipeXYZCommand`.

**Review by rizemon — CHANGES_REQUESTED** [review]
Some changes

**Review by rizemon — APPROVED** [review]
LGTM

**Pull request merged**